packages/wdio-config/src/types.ts

```diff
@@ -6,16 +6,19 @@ export type Hooks = {
 export type Capabilities = (WebDriver.DesiredCapabilities | WebDriver.W3CCapabilities)[] | WebdriverIO.MultiRemoteCapabilities;
 
 export interface ConfigOptions extends Omit<WebdriverIO.Config, 'capabilities' | keyof WebdriverIO.Hooks>, Hooks {
+    spec?: string[];
+    suite?: string[];
     capabilities?: Capabilities;
     specFileRetryAttempts?: number;
+    cucumberFeaturesWithLineNumbers?: string[];
```

Comment:
It seems we have missed to add this to the docs. Could we add this to the [testrunner options](https://github.com/webdriverio/webdriverio/blob/master/docs/Options.md#testrunner-options)?

Reply:
OK, sure I'll add that to the doc. Let me know what should be the one-line description for the same.